test/test_simulation_results.jl

```diff
@@ -218,14 +218,22 @@ function test_simulation_results(file_path::String, export_path)
             len = 10,
         )[:P__ThermalStandard])[1] == 10
 
-        # request bad window
+        # request good window
         @test_throws IS.InvalidValue read_realized_variables(
             results_ed,
             names = [:P__ThermalStandard],
             initial_time = DateTime("2024-01-02T23:10:00"),
             len = 11,
         )
 
+        # request bad window
+        @test_throws IS.InvalidValue read_realized_variables(
```

Comment:
Does this cause an expected error log message to get printed to the console? If so, can you wrap this statement in a `@test_logs`?